### ISSUE TITLE
TODO: Adding the bot commands to the readme or somewhere

### ISSUE THREAD

**Issue by OhMyGuus** [OhMyGuus/I-Still-Dont-Care-About-Cookies#10700]
I've created a bot so people can help me out a bit & so the pull requests are easier to maintain.

Commands in issues:
- /overlay -> Adds the `found` label & `overlay-issue` label
- /scrollbar -> Adds the `found` label & `scollbar-issue` label
- /found -> Adds the `found` label
- /notfound -> Adds the `notfound` label
- /fixed -> Adds the `fixed` label
- /blocked -> Add the `website-blocked` label
- /nsfw-> Add the `NSFW` label

Things for in pullrequests:
{auto_update} ->Adds all fixed sites + the linked issues under this prefix ( Gathered from the commit messages & comments, body above this) -> Will remove everything's under this so write only things above this prefix.
{bot_add_items} -> Add this to the comment and it will grab the sites in the message  and adds it from {auto_update}
{bot_remove_items} -> Add this to the comment and it will grab the sites in the message and removes it from {auto_update}

--- 
currently everyone can use these commands but if there is abuse I will block the user from it. 

-- 

For any other command suggestions let me know 👍

**Comment by inson1** [comment]
webpage blocked?

**Comment by OhMyGuus** [comment]
> webpage blocked?

Good one added it with nsfw as well 👍

**Comment by inson1** [comment]
limit for websites per PR would maybe be good? - 150?

also good idea to readme

> To install the dev build:
> Download it from
> https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/actions/runs/7366943876
> and then click on ISDCAC-chrome
> This will download a zip that you can move into the chrome://extensions tab if you enable developer mode on the right top corner.

screenshot when responding to notfound

**Comment by inson1** [comment]
Is there way to make some slower processes faster?

**Comment by inson1** [comment]
What about the problem with cookies not on main page, but on secondary page? bot is not taking full route

**Comment by inson1** [comment]
blocked should add also found

**Comment by inson1** [comment]
maybe it would be good to say differences between fixed and notfound

**Comment by inson1** [comment]
What about not on main page label and command?

**Comment by inson1** [comment]
Is important to check all browsers?

**Comment by inson1** [comment]
found should remove notfound or fixed

**Comment by inson1** [comment]
it doesnt feel like the bot works fast andproperly

**Comment by OhMyGuus** [comment]
> it doesnt feel like the bot works fast and properly

yeah there seems to be a issue in it still that breaks the bot once in a while, need to look into
